### PR TITLE
Create Sahelanthropus_dificulty.md

### PR DESCRIPTION
**Pull request by TheHuntingParty**
This pull request has been automatically generated by prose.io.

### CODE REVIEW

File: wiki/Sahelanthropus_dificulty.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+	PTLB 	=	240,		 
+	PTRB 	=	240,		 
+	}
+    
+The health points on each part, defines when that same part explodes during the boss fight, when the health points drop to 0.
+
+## Defining Sahelanthropus Attack damage and speed
+
+Sahelanthropus Attack damage and speed are controled by **TYPE_OKB** Variable, example:
+
```
Comment (mgsvmoddingwiki) [inline wiki/Sahelanthropus_dificulty.md:71]: Add
```lua

Comment (mgsvmoddingwiki) [inline wiki/Sahelanthropus_dificulty.md:71]: Add a newline between the bottom of the text line and then add \`\`\`lua

File: wiki/Sahelanthropus_dificulty.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
+---
+
```
Comment (mgsvmoddingwiki) [inline wiki/Sahelanthropus_dificulty.md:2]: Remove blank lines on top and bottom between \-\-\- and the 3 frontmatter lines

**Review by TheHuntingParty — COMMENTED** [review]
done!

**Review by mgsvmoddingwiki — COMMENTED** [review]
Done

**Pull request merged**